infrastructure/keymanager/src/lib.rs

```diff
@@ -0,0 +1,14 @@
+extern crate crypto;
```

Comment:
use edition 2018, and you dont need these extern crates

Reply:
Change already applied from previous comment